src/courtroom.cpp

```diff
@@ -4032,28 +4030,32 @@ void Courtroom::set_sfx_dropdown()
     ui_sfx_remove->hide();
     return;
   }
-  QStringList soundlist = ao_app->get_list_file(
+  sound_list = ao_app->get_list_file(
       ao_app->get_character_path(current_char, "soundlist.ini"));
 
-  if (soundlist.size() <= 0) {
-    soundlist = ao_app->get_list_file(
+  if (sound_list.size() <= 0) {
+    sound_list = ao_app->get_list_file(
         ao_app->get_theme_path("character_soundlist.ini"));
-    if (soundlist.size() <= 0) {
-      soundlist = ao_app->get_list_file(
+    if (sound_list.size() <= 0) {
+      sound_list = ao_app->get_list_file(
           ao_app->get_default_theme_path("character_soundlist.ini"));
```

Comment:
Why do we store this in the theme folder instead of the base folder like the user-defined iniswaps INI?

Reply:
🤔 💭

Reply:
i have adressed this